src/SUMMARY.md

```diff
@@ -40,12 +40,13 @@
       - [iwd](./config/network/iwd.md)
       - [NetworkManager](./config/network/networkmanager.md)
       - [ConnMan](./config/network/connman.md)
-   - [Graphics](./config/graphics/index.md)
-      - [Intel GPU](./config/graphics/intel.md)
-      - [NVIDIA Optimus](./config/graphics/optimus.md)
-   - [Xorg](./config/xorg/index.md)
-      - [Session Management](./config/xorg/session-management.md)
-   - [Wayland](./config/wayland.md)
+   - [Graphical Session](./config/graphical-session/index.md)
+      - [Graphics](./config/graphical-session/graphics/index.md)
```

Comment:
This could probably be 'Graphics Drivers' at this point

Reply:
Agreed